File: RandomNumber/RandomNumber/ViewController.m
//
//  ViewController.m
//  RandomNumber
//
//  Created by Michael Balsa on 5/29/22.
//

#import "ViewController.h"

@interface ViewController ()







@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}


- (IBAction)numberGenerator:(id)sender {
    int min = 1;
    int max =55;
    
    int randomNumber = arc4random_uniform(max - min) + min;
    self.labelTwo.text = [NSString stringWithFormat: @"%i", randomNumber];
}

- (IBAction)number:(id)sender {
    int randomNumber = arc4random() % 55;
    //starts from 0 to 55.
    self.labelOne.text = [NSString stringWithFormat: @"%i", randomNumber];
    //Allows the label to convert integer value to string value
}
@end

File: Time Tables/Time Tables/ViewController.swift
//
//  ViewController.swift
//  Time Tables
//
//  Created by Michael Balsa on 9/19/21.
//

import UIKit

class ViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
    
    @IBOutlet var Slider: UISlider!
    
    @IBOutlet var table: UITableView!
    @IBAction func SliderChange(_ sender: Any) {
        table.reloadData()
        
    }
    internal func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
        return 50 
    }
    
    internal func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
        let cell = UITableViewCell(style: UITableViewCell.CellStyle.default, reuseIdentifier: "Cell")
        cell.textLabel?.text = String((Int(Slider.value * 20)) * (indexPath.row + 1))
        return cell
    }
    
        
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }


}

